### ISSUE TITLE
Unable to disable shellcheck with `SHELLCHECK_PATH=""`

### ISSUE THREAD

**Issue by b0o** [bash-lsp/bash-language-server#570]
#555 seems to have broken the functionality allowing shellcheck to be disabled by setting the environment variable `SHELLCHECK_PATH` to the empty string. See [here](https://github.com/bash-lsp/bash-language-server/blob/938bf337ec6afb7b909167165e03759a65a751d4/server/src/config.ts#L5) and #432.

Specifically, I think this change was the cause:
```diff
diff --git a/server/src/linter.ts b/server/src/linter.ts
index 936341a3..6cb7087c 100644
--- a/server/src/linter.ts
+++ b/server/src/linter.ts
@@ -1,6 +1,9 @@
 import { spawn } from 'child_process'
 import * as LSP from 'vscode-languageserver'
 
+import * as config from './config'
+import { execShellScript } from './util/sh'
+
 function formatMessage(comment: ShellcheckComment): string {
   return (comment.code ? `SC${comment.code}: ` : '') + comment.message
 }
@@ -10,8 +13,23 @@ type LinterOptions = {
   cwd?: string
 }
 
-export default class Linter {
-  private executablePath: string | null
+export async function getLinterExecutablePath(): Promise<string | null> {
+  const pathFromConfig = config.getShellcheckPath()
+
+  if (pathFromConfig) {
+    return pathFromConfig
+  }
+
+  try {
+    const path = (await execShellScript('which shellcheck')).trim()
+    return path === '' ? null : path
+  } catch (e) {
+    return null
+  }
+}
+
+export class Linter {
+  public executablePath: string | null
   private cwd: string
   _canLint: boolean
```

**Comment by skovhus** [comment]
Thanks for reporting this.

**Comment by skovhus** [comment]
This has been fixed in bash-language-server 3.3.0 and in the latest VSCode extension. Let me know if this doesn't work for you. :)

**Comment by b0o** [comment]
That fixed it, thank you very much!
